### ISSUE TITLE
Are libfirm and cparser still alive? Are there people out there interested in keeping it alive?

### ISSUE THREAD

**Issue by rochus-keller** [libfirm/libfirm#36]
Firm and Cparser are indeed impressive and useful works. Im currently reading the papers, theses and reports available on the web and wonder why there isn't more activity here on Github.

**Comment by rochus-keller** [comment]
The contact email firm@ipd.info.uni-karlsruhe.de on the Firm homepage is obviously dead (Diagnostic-Code: X-Postfix; Host or domain name not found. Name service error for name=ipd.info.uni-karlsruhe.de type=A: Host not found).

**Comment by mtk2xfugaku** [comment]
Yeah it is a great tool for learning about optimizing compilers beyond a toy implementation since the core back-end is still relatively small compared to something like LLVM, etc. From my last few pull request I have made it work with Python >=3.5 but haven't had enough time to make any serious contribution yet (I am planning on writing some blogpost about libfirm and some more contribution).  Beside LLVM, libfirm is the only good enough compiler infrastructure for learning.

**Comment by rochus-keller** [comment]
Thanks. Yes, it has an impressive architecture and modular structure. But unfortunately the optimizer seems to have quite little effect (see https://github.com/libfirm/libfirm/issues/37), so eventually I'm just using cparser which I currently retarget to the Eigen backend (see https://github.com/rochus-keller/EiGen/tree/cparser/cparser).

**Comment by midipix** [comment]
Interested in keeping it alive and completing libfirm's support of the x64 (as opposed to the canonical amd64) semantics.
